### ISSUE TITLE
The instructions for connecting a camera are misleading for Pi5

### ISSUE THREAD

**Issue by sandyol55** [raspberrypi/documentation#3915]
In the "Accessories/Camera/Connect the Camera" section the current instructions are :-

_The flex cable inserts into the connector labelled CAMERA on the Raspberry Pi, which is located between the Ethernet and HDMI ports. The cable must be inserted with the silver contacts facing the HDMI port._

The 2nd is incorrect for Pi5, and neither are appropriate for the Zero  range.

Propose some re-wording referencing the alignment of the cable stiffener relative to the connector latch rather than the surrounding components, and perhaps an image as below.

Happy to have a go at a PR.

![Camera_Con2](https://github.com/user-attachments/assets/3164d8cc-517e-49f2-b3b5-a5663418fa1b)

**Comment by nathan-contino** [comment]
Agreed that this could use some clarification. I've found myself looking for this exact doc in the last couple of weeks, so I know it's not covered right now.

Please do open a PR with some fixes, even if it's a little rough. Happy to clean it up and get it over the finish line.

**Comment by robertpeteuil** [comment]
Came here to report this issue.

I just wasted two hours troubleshooting my camera connection because all the docs/guides/videos are wrong for the Raspberry Pi 5.

**Comment by nathan-contino** [comment]
We hear you, @robertpeteuil . I've experienced similar problems in the past. I'll take some better pictures of this tomorrow so we can get this fixed. For too long I've allowed perfect to be the enemy of good here.

**Comment by lurch** [comment]
@nathan-contino The linked PR has now been auto-closed?

**Comment by nathan-contino** [comment]
I'll be adding our own pictures that are more in line with our style guide. But we do appreciate the contribution!

**Comment by katshann-raspberry** [comment]
Fixed by latest updates to the camera article.
